ts/features/idpay/initiative/details/store/index.ts

```diff
@@ -107,7 +107,7 @@ export const idpayPaginatedTimelineSelector = createSelector(
   initiative => initiative.timeline
 );
 
-export const idpayTimelineSelector = createSelector(
+export const idpayOperationListSelector = createSelector(
```

Comment:
We used everywhere the word `timeline` to refer to the operation list of an initiative. So why change this here?

Reply:
`OperationList` is a second-level value in `timeline`, so the selectors prefixed with `timeline` return values from its root, while the ones prefixed with `OperationList` return  a non-po t `timeline.operationList`.

I thought of `timeline` as the whole data union that refers to a timeline as in {pageCount, lastUpdate, operationList ...etc}

this way, the `timeline`-prefixed selectors are mostly used to manipulate and check the Pot value, while `operationList` ones are used for direct rendering purposes and at first glance in a component it is (imho, although i am probably in a fallacy since I wrote it) easier to understand and cleaner to read

Reply:
That makes sense! I like it.